app/src/components/attachments/__snapshots__/DropZone.test.tsx.snap

```diff
@@ -39,6 +39,11 @@ exports[`DropZone matches the snapshot 1`] = `
             Browse Files
           </a>
         </h3>
+        <span
+          class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
+        >
+          Accepted file types: undefined
```

Comment:
we should pass in acceptedFileExtensions as a dropZoneProps here so that the snapshot doesnt say undefined